ios/ApplePayButtonManager.swift

```diff
@@ -3,9 +3,7 @@ import Foundation
 @objc(ApplePayButtonManager)
 class ApplePayButtonManager: RCTViewManager {
     override func view() -> UIView! {
-        let view = ApplePayButtonView(frame: CGRect.init())
-        view.stripeSdk = bridge.module(forName: "StripeSdk") as? StripeSdk
-        return view
+        return ApplePayButtonView(frame: CGRect.init())
```

Comment:
Since the stripe sdk implementation is now a singleton it doesn't need to be passed around from the native module.